ui/test/shared/components/TableGraph.test.tsx

```diff
@@ -44,7 +45,7 @@ const setup = (override = []) => {
 
 describe('Components.Shared.TableGraph', () => {
   describe('functions', () => {
-    describe('filterInvisibleColumns', () => {
+    describe('filterTableColumns', () => {
```

Comment:
I think all of these tests should be removed from TableGraph.test.tsx and moved into the timeSeriesToDygraph.test file since thats where what is being tested is located in now.

Reply:
ooh good idea. thanks!